app/models/items/donation.rb

```diff
@@ -12,7 +12,8 @@ def self.default_code
   validates_presence_of :account_code_id
   
   belongs_to :customer
-  
+  belongs_to :recurring_donation, class_name: 'RecurringDonation', foreign_key: :recurring_donation_id
```

Comment:
i don't think you need `class_name` and `foreign_key` here since Rails will correctly derive them from the owning class's name.  unless i've misunderstood, please delete those

Reply:
changed